### ISSUE TITLE
Freezes at or right after landing

### ISSUE THREAD

**Issue by scottykarate1** [fhorinek/BB#369]
Hardware Version 22
Firmware version 278

I have flown with it 3 times on FW 278. All 3 times it froze after landing.  Everything works while flying. IGC file show full flights.  Have to hard reset it.
Another minor issue is Flightbook > drill to flight IGC file. When I try the "Show on Map" option,  the device will show a strange grid line pattern and not go any further.  The back button will take you back to the flight log stats page.

**Comment by bubeck** [comment]
Thanks for your bug report.

Could you please send the igc file which gives problems showing it?
If possible, could you please also send the debug.log stored on the Strato? Read https://github.com/fhorinek/BB/blob/master/Dev/Logging.md if you need to know how to turn debugging on.

Are the maps installed for the area you are flying in?

**Comment by scottykarate1** [comment]
[Debug & IGC.zip](https://github.com/fhorinek/BB/files/9699437/Debug.IGC.zip)
Here is the latest IGC. Showing any IGC gives the problem The debug looks like it is from a device crash when I tried to view a flight that was logged before 278 was installed and it crashed when the analyzing the igc. 
Maps were installed prior to updating to 278.  Should they be reinstalled? They still work on the map screen on the device.

**Comment by bubeck** [comment]
I will take a look and give you results. If map is working, there is no need to reinstall.

**Comment by Tokter** [comment]
I reset/formatted my Strato. Then updated the bootloader, then copied the 278 firmware.
It seems to work fine, but when I plug it into my computer:
- The screen switches to: Charging.
- When I open the Strato node in windows explorer, the screen changes to: USB mode
- Then when I open "Internal Storage", the loading animation on the Strato screen freezes, and the explorer never returns any files.
It looks like the Strato froze and I have to use Left/Right buttons to reset.

**Comment by bubeck** [comment]
There is a BIG difference between D269 and newer and all previous R-versions. Newer versions (like D278) use MTP to make a computer access the files. This is the same protocol as used by android phones. Before that a raw USB block device was exposed, similar to a USB stick. 

This change means, that the Strato must be reformatted by the firmware, all data is lost, and there is no way back. From what version did you update? Is your computer able to use MTP to access a device? Are you able to access a android phone with your computer?


> I reset/formatted my Strato. Then updated the bootloader, then copied the 278 firmware. It seems to work fine, but when I plug it into my computer:
> 
>     * The screen switches to: Charging.
> 
>     * When I open the Strato node in windows explorer, the screen changes to: USB mode
> 
>     * Then when I open "Internal Storage", the loading animation on the Strato screen freezes, and the explorer never returns any files.
>       It looks like the Strato froze and I have to use Left/Right buttons to reset.

**Comment by Tokter** [comment]
_From what version did you update?_

It might have been D269. I updated over wifi to D278. But it would always freeze 2-3 seconds after turning on. Which was not enough time to try anything.
So I used the 3-Button method to wipe/format the device, update the bootloader and copied D278 over the USB.
It no longer freezes after turning on, but when you try to access it via USB.

_Is your computer able to use MTP to access a device? Are you able to access a android phone with your computer?_
Yes, my Pixel phone works fine. Which uses MTP. I also tested with multiple computers, Windows 10 and Windows 11.

Just wanted to report it, I'll try it again once a new build is out. Thanks!
